### PR TITLE
Simplified getting started, fix page structure

### PR DESCRIPTION
**Pull request by skim1420**
@duftler @rbuskens resubmitting

### CODE REVIEW

**Comment by duftler** [comment]
@tomas PTAL

File: documentation/getting_started.md [hunk 1]
```diff
@@ -177,54 +169,43 @@ Create an AWS virtual machine.
...
 
-If you have chosen to run Spinnaker inside an Amazon Web Services or
-Google Compute Engine VM, please ssh into the VM.
+If you have chosen to run your Spinnaker instance somwhere other than on AWS or GCP (e.g. your local workstation),
+follow these steps:
 
-Irrespective of the machine on which you have chosen to run Spinnaker,
-at the command prompt, type in (or cut-and-paste) the following
-command:
+If you just want to try running on your local workstation, type in the following command:
```
Comment (duftler) [inline documentation/getting_started.md]: Is this line still necessary?

**Comment by duftler** [comment]
LGTM after addressing the comment.

File: documentation/getting_started.md [hunk 2]
```diff
@@ -10,26 +10,21 @@ lang: en
...
+1. Set up an Amazon Web Services (AWS) EC2, Google Compute Engine instance 
+or local machine that will be used to run Spinnaker.
 1. Configure an example Spinnaker pipeline to bake an image and deploy
 the image to a cluster, and let you explore Spinnaker in operation.
 1. When you're done experimenting, clean everything up and stop Spinnaker.
 
-The first two steps, which have very little to do with Spinnaker
-itself, are by far the most complex steps.
-
 Keep in mind that naming of your entities in AWS is important as these names are parsed
```
Comment (tomaslin) [inline documentation/getting_started.md]: I know it's unrelated to this PR, but line 28 feels like it belongs in the AWS configuration section

Comment (skim1420) [inline documentation/getting_started.md]: I decided to cater to tunnel vision, and add this note (give spinnaker some time to configure itself) to every setup path (aws, gcp, local).

**Comment by tomaslin** [comment]
Changes look good aside from comments

File: documentation/getting_started.md (hunk 1 above)
Comment (skim1420) [inline documentation/getting_started.md]: Yep, removed the first one.

**Comment by skim1420** [comment]
Let's revisit min specs on instance types in a different PR.

**Pull request merged**